src/main/java/de/tum/in/www1/artemis/service/connectors/localci/dto/LocalCIBuildJobQueueItem.java

```diff
@@ -108,6 +111,14 @@ public void setCourseId(long courseId) {
         this.courseId = courseId;
     }
 
+    public boolean isTestPush() {
+        return isTestPush;
+    }
```

Comment:
Might just be me, I would not have understood what it refers too without further knowledge

I would prefer the name `isPushToRepository` and a javadoc comment here on which implication come with the test repository _(the reason why we need this variable)_

Reply:
done